config/spark-fairscheduler.xml

```diff
@@ -0,0 +1,13 @@
+<allocations>
+    <pool name="default">
+        <schedulingMode>FAIR</schedulingMode>
+        <weight>1</weight>
+        <minShare>1</minShare>
+    </pool>
+
+    <pool name="highPriority">
```

Comment:
Can any user choose the high priority pool?

Reply:
yea. Currently yes. Maybe we can configure some user grouping in the future. But that's up to a discussion.

Reply:
Ok, as long as Gazi is ok with it, I'm ok with it

Reply:
I think it's unrelated to this PR. Users can request all available resources currently. We will for sure implementing something that allow/deny users to request high priority queues. But not now.

Reply:
Same response as above